backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/InformQueueService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using EasyMeets.Core.BLL.Interfaces;
+using EasyMeets.RabbitMQ.Interface;
+
+namespace EasyMeets.Core.BLL.Services
```

Comment:
You need to dispose connection

Reply:
made Dispose inside ProducerService ( because use ProducerService instance here )  , its enough ?